main.go

```diff
@@ -60,12 +60,47 @@ func sortedKeys(m map[string]struct{}) []string {
 	return keys
 }
 
+func parseOverrideDependencies(s string, dependencies []string) map[string]string {
+
+	// parsing through the list of overrides to make an original:new string map
+	var overrideDependenciesList []string
+	if len(s) > 3 {
+		overrideDependenciesList = strings.Split(s, ",")
+	}
+	overrideDependenciesMap := make(map[string]string)
+
+	for _, s := range overrideDependenciesList {
```

Comment:
you are overriding `s` variable. `s` is all the input to this function. Not really a bug in this case but it is still confusing. This is why it is advised to give variables contextual names